src/app/tabs/tabs-module.demo.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-tabs-module-demo',
+  templateUrl: '/tabs-module.demo.html'
+})
+export class TabsModuleDemoComponent {
+
+  data: Array<any> = [
```

Comment:
The new ivy compiler will require these to be public, so would make sense to start getting into that frame of mind.

Reply:
Oh really, so any template binding requires the component variable to be public?

Reply:
Unless things have changed again, any member required in the associated template should be public.  There was a long discussion on the angular forums about this.

Whilst trying to get the demo app running with IVY (see ivy branch) I had to make all these public otherwise I get "not found" errors, in fact I think it was excluding them.